internal/grpchelper/grpchelper.go

```diff
@@ -0,0 +1,58 @@
+// Package grpchelper implements helper functions to be used with slam service grpc clients
```

Comment:
My understanding is this will be deleted in favor of  using https://github.com/viamrobotics/rdk/blob/bec73d4cef679ab41f877dd0696a5b0d5ab08122/services/slam/grpchelper/grpchelper.go in a follow-up PR

Reply:
Yes